apps/web/src/server-setup.ts

```diff
@@ -1,15 +1,17 @@
-import { setupBlitz } from "@blitzjs/next";
-import { AuthServerPlugin, PrismaStorage } from "@blitzjs/auth";
-import { prisma as db } from "../prisma/index";
+import {setupBlitz} from "@blitzjs/next"
+import {AuthServerPlugin, PrismaStorage} from "@blitzjs/auth"
+import {prisma as db} from "../prisma/index"
 
-const { withBlitz, gSSP, api } = setupBlitz({
-  plugins: [AuthServerPlugin({
-    storage: PrismaStorage(db),
-    isAuthorized() {
-      console.log("isAuthorized");
-      return false;
-    }
-  })]
+const {withBlitz, gSSP, gSP, api} = setupBlitz({
```

Comment:
I think `withBlitz` should only be exported from setupClient?

Reply:
Yep. It was previously what `api` is now (wrapper for API handlers). Will remove it